### ISSUE TITLE
OSVOS-S

### ISSUE THREAD

**Issue by bhack** [scaelles/OSVOS-TensorFlow#8]
Do you plan to release the new OSVOS-S with instance segmentation?

**Comment by scaelles** [comment]
Thanks a lot for your interest in our work! We will publish the code upon acceptance of the paper.

**Comment by bhack** [comment]
Where was submitted?

**Comment by scaelles** [comment]
We have submitted it to a journal and we are waiting for their reviews.

**Comment by bhack** [comment]
I cannot see in the paper if you have experimented the fine-tuning of the instance segmentation mask  network using the GT of the 1st frame.

**Comment by scaelles** [comment]
In the paper we use the instance segmentation network out of the box without any fine-tuning.

**Comment by bhack** [comment]
Yes I've seen.. but have you experimented the fine-tuning? Degraded?

**Comment by scaelles** [comment]
In DAVIS 2016, instances from different classes are mixed in the same mask so it would be difficult to use the 1st frame gt mask to fine-tune a specific class of the instance segmentation network.

**Comment by bhack** [comment]
But Davis 2017 could not have this problem

**Comment by scaelles** [comment]
It could be, but we haven't done any experiments in that direction.

**Comment by bhack** [comment]
So, next extension :)
